pkg/tfgen/edit_rules.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2016-2022, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tfgen
+
+import (
+	"bytes"
+	"fmt"
+	"path/filepath"
+	"regexp"
+
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+)
+
+func defaultEditRules() editRules {
+	return editRules{
+		// Replace content such as "`terraform plan`" with "`pulumi preview`"
+		boundedReplace("[tT]erraform [pP]lan", "pulumi preview"),
+		// Replace content such as " Terraform Apply." with " pulumi up."
+		boundedReplace("[tT]erraform [aA]pply", "pulumi up"),
+		reReplace(`"([mM])ade (by|with) [tT]erraform"`, `"Made $2 Pulumi"`),
```

Comment:
Adding this rule is the actual change.

Reply:
Do we feel confident that capitalizing the replacement will make sense in all situations?

Reply:
I think so. It's only for quoted strings, so I think this will only happen in description fields where it would make sense.